manifests/params.pp

```diff
@@ -55,6 +56,10 @@
     $maintenance_mode         = hiera('wso2::maintenance_mode')
     $install_mode             = hiera('wso2::install_mode')
 
+    $config_mount             = hiera('wso2::config_mount')
+    $gov_mount                = hiera('wso2::gov_mount')
```

Comment:
Can't these be re-used from the `$registry_mounts`?

Reply:
Modified to re-use $registry_mounts